modules/hp_timeline/templates/paragraph--hp-timeline-field.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="entry">
+    <div class="title">
+        <h3>{{ content.field_hp_timeline_field_datetime[0] }}</h3>
+            {% if content.field_hp_timeline_field_title[0] is not null %}
+                <p>{{ content.field_hp_timeline_field_title[0] }}</p>
```

Comment:
is the 0 needed here? why isnt it just {{ {{ content.field_hp_timeline_field_title }} ?